packages/wonder-blocks-core/util/enumerate-scroll-ancestors.ts

```diff
@@ -31,7 +31,7 @@ const canScroll = function(node: Element) {
 // iterator property itself. $FlowFixMe
 class ScrollAncestorsIterator implements Iterator<Element> {
     done: boolean = false;
-    parentElement: ?Element;
+    parentElement: Element | null;
```

Comment:
This isn't actually necessary.  I didn't realize that `Element` was a nullable type and I had to turn on strict null checks to get the safety.